client/ayon_core/hosts/houdini/plugins/publish/extract_hda.py

```diff
@@ -26,6 +26,14 @@ def process(self, instance):
         hda_def.setOptions(hda_options)
         hda_def.save(hda_def.libraryFilePath(), hda_node, hda_options)
 
+        if instance.data["creator_attributes"].get("use_project"):
+            # Set TAB Menu location interactively
+            # This shouldn't be needed if the Tool Location is saved in the HDA.
+            tool_name = hou.shelves.defaultToolName(
+                hda_def.nodeTypeCategory().name(), hda_def.nodeTypeName())
+            hou.shelves.tool(tool_name).setToolLocations(
+                ("AYON/{}".format(self.project_name),))
```

Comment:
Encounter this error with this line of code.
```
Traceback (most recent call last):
  File "C:\Users\Kayla\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\ayon-core\client\ayon_core\hosts\houdini\plugins\publish\extract_hda.py", line 35, in process
    ("AYON/{}".format(self.project_name),))
AttributeError: 'ExtractHDA' object has no attribute 'project_name'
```
Consider this to fix the bug.
```suggestion
            project_name = instance.context.data["projectName"]
            hou.shelves.tool(tool_name).setToolLocations(
                ("AYON/{}".format(project_name)))
```